src/compile-middleware.ts

```diff
@@ -66,6 +67,13 @@ function isSuccessful(response: Response) {
   return (statusCode >= 200 && statusCode < 300);
 }
 
+// NOTE: To change the max length of the cache at runtime, just use bracket
+// notation, i.e. `babelCompileCache['max'] = 64 * 1024` for 64KB limit.
+export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
```

Comment:
this notation is a little crazy, why is the `<LRU.Options<string>>` cast needed? Is that needed to denote that you are configuring a cache of strings?